lhotse/cut.py

```diff
@@ -39,6 +39,16 @@ class Cut:
     """
     A Cut is a single "segment" that we'll train on. It contains the features corresponding to
     a piece of a recording, with zero or more SupervisionSegments.
+
+    The SupervisionSegments indicate which time spans of the Cut contain some kind of supervision information:
+    e.g. transcript, speaker, language, etc. The regions without a corresponding SupervisionSegment may
+    contain anything - usually we assume it's either silence or some kind of noise.
+
+    Note: The SupervisionSegment time boundaries are relative to the beginning of the cut.
+    E.g. if the underlying Recording starts at 0s (always true), the Cut starts at 100s,
+    and the SupervisionSegment starts at 3s, it means in the Recording, the supervision actually started at 103s.
```

Comment:
I'd rephrase to "it means that in the Recording the..."
the sentence doesn't read well currently.